doc/README.md

```diff
@@ -1,56 +1,16 @@
-
-As a side effect of the necessity of these mappings, we will also gain offline visibility in to what data is available for collection and the ability to easily explore the data. Further, we can begin analyzing data coverage, etc. Even more importantly, we can begin to validate the data on a cross-platform basis! There are huge quality assurance benefits to having an offline vision for our platform data.
-
-In order to solve these problems, we must first understand all of the difficulties and arcane methods to get this data, and provide easy presentation and usage. *yay*
-
-## Components
-
-![Architecture](/doc/img/tdm_arch.png)
-
-TDM is made up of several different technologies. These are expanded in their own code sub-directories if not here.
-* [ETL Code](/etl/)
-* [Web Code](/web/)
-* [NGINX / Goaccess](/nginx/)
-* [ArangoDB](https://www.arangodb.com/)  
-ArangoDB was being used elsewhere in our team and thus we decided to bring it in to another project. Being a graph database, it had some attractive features, flexibility, and GUI that could in theory make it easier for relatively technical users to directly use the database instead of building custom web interfaces. ArangoDB contains our processed source of truth in TDM. The ETL process parses all of the different data we are interested in and formalizes relationships into ArangoDB to enable querying capabilities on the data instead of doing all the work in code. ArangoDB has a nice query language for traversing schemas, as it is a graph database, and simplifies some queries. However, it does come with some limitations in maturity such as lacking "views" (now implemented*), etc. We are currently using it very much like a relational database, and should likely eventually transition to an RDBMS. However, given it works and having operationalized around it, there's no pressing need to do so.
-* [Elasticsearch](https://www.elastic.co/products/elasticsearch)  
-Elasticsearch is effectively a mitigation of search difficulty in ArangoDB. In order to achieve reasonable performance, we wrote a query which resulted in monstrous RAM usage (32 GB for "bgp") with potential to take up to a minute to return. This was unacceptable. We could revisit aspects of TDM's design, but decided to try out ES. Elasticsearch receives all of the data in TDM in a denormalized form to enable fast and powerful search capabilities. Elasticsearch is perfectly suited to our search needs, and solved a couple search issues at once by implicitly handling fuzziness, scoring of results, and more. Our current usage is by no means perfect and is rather naive, but it's extremely fast and sits at a near constant 1 GB of RAM usage during search loads.
```

Comment:
Hmm interesting perspective. From the developer standpoint versus the user standpoint, do you appreciate that honesty and thought or prefer purely capability oriented doc? These issues could be exposed in issues instead, and they are unnecessary.

Reply:
Hm. As a developer I personally prefer purely capability oriented documentation bc I'm trying to weed through enough as it is to figure out exactly what it is I'm looking for. The honesty definitely plays a part in building a credible relationship, but maybe it could be phrased differently -- rather than "Our current usage is by no means perfect and is rather naive, but it's extremely fast and sits at a near constant 1 GB of RAM usage during search loads" something more along the lines of "Our simple, but extremely fast usage sits at a near constant 1 GB of RAM..". Same for the lines where you describe the Web UI as messy